src/bin/casc/args.rs

```diff
@@ -1,13 +1,25 @@
 use clap::Parser;
 
 #[derive(Parser, Debug)]
-#[clap(author, version, name = "casc", about = "Compile Cascade SELinux policies into CIL", long_about = None)]
+#[clap(
+    author,
+    version,
+    name = "casc",
+    about = "Compile Cascade SELinux policies into CIL",
+    long_about = "Compile Cascade SELinux policies into CIL
+Supported use cases:
```

Comment:
Thanks for updating this way.  Upon seeing it, I think I'm reconsidering the list approach at all.

I think I'd rather push as much description as possible down to the OPTIONS section below.  Currently a fair amount is duplicated between DESCRIPTION and OPTIONS.

I think what might make more sense is a first paragraph saying "Compile..." just like now.  Then a second paragraph that explains that the -o and -s options are mutually exclusive and referring the user to the OPTIONS section.